src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/WasmTestCommand.cs

```diff
@@ -18,16 +22,25 @@ internal class WasmTestCommand : TestCommand
         protected override TestCommandArguments TestArguments => _arguments;
         protected override string CommandUsage { get; } = "wasm test [OPTIONS]";
```

Comment:
Please update this to match the `--` extra args

Reply:
will update with `wasm test [OPTIONS] -- [ENGINE OPTIONS]`